inflate_bench_test.go

```
@@ -0,0 +1,158 @@
+
+func inflatePackZlibStreaming(src, dst []byte) (int, error) {
+	if len(src) < 2 {
+		return 0, errDeflateBadData
+	}
+	if err := validateZlibHeader(src[0], src[1]); err != nil {
+		return 0, err
+	}
+
+	br := getBytesReader(src[2:])
+	fr := flatePool.Get().(io.ReadCloser)
+	if err := fr.(flate.Resetter).Reset(br, nil); err != nil {
```

golangci-lint (2.12.2)

[error] right hand must be only type assertion

(forcetypeassert)